test/export/test_export.py

```diff
@@ -670,16 +670,23 @@ class MyDataClass:
         roundtrip_spec = treespec_loads(treespec_dumps(spec))
         self.assertEqual(roundtrip_spec, spec)
 
+        @dataclass
+        class MyOtherDataClass:  # the pytree registration don't allow registering the same class twice
+            x: int
+            y: int
+            z: int = None
+
```

Comment:
Did you mean to put this in the next PR up when we unify python and C++ pytree registration?

Reply:
I have moved this to the next PR.